code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -1,4 +1,22 @@
-#define PUNCH_STAMINA_MULTIPLIER 2.6
+#define TRAIT_ALREADYSTAMINAFLOORED    	"alreadystaminafloored" //Trait to keep track whether someone was floored recently by stamina damage.
+#define STAMINA_KNOCKDOWN_COOLDOWN 10 SECONDS
+
+#define MAX_HUMAN_LIFE 135
```

Comment:
Something is sussy about this entire files edits.

Reply:
There was a cleanup that took place that removed a lot of the old defines between the original PR and the upstream merge, I missed it.